base/sysimg.jl

```diff
@@ -84,6 +84,8 @@ include("iterator.jl")
 include("osutils.jl")
 
 # strings & printing
+include("utferror.jl")
+include("utftype.jl")
```

Comment:
totally superficial, sorry... I think this would be a tiny bit better as `utftypes.jl` ?

p.s. thank you thank you thank you for opening this separately. :+1: